api/organisations/tests/test_models.py

```diff
@@ -114,7 +114,11 @@ def test_organisation_over_plan_seats_limit_returns_true_if_over_plan_seats_limi
 
 def test_organisation_over_plan_seats_no_subscription(organisation, mocker, admin_user):
     # Given
-    mocker.patch("organisations.models.MAX_SEATS_IN_FREE_PLAN", 0)
+    mocker.patch(
```

Comment:
We don't need to mock this - `max_seats` is a concrete field on the subscription class, we can just do 

```python
organisation.subscription.max_seats = 0
organisation.subscription.save()
```

Reply:
Ok thanks, done.